sonar-plugin-api/src/test/java/org/sonar/api/utils/System2Test.java

```diff
@@ -71,12 +79,12 @@ public void testEnvVariable() throws Exception {
   }
 
   @Test
-  public void testIsOsWindows() throws Exception {
+  public void testIsOsWindows() {
     assertThat(System2.INSTANCE.isOsWindows()).isEqualTo(SystemUtils.IS_OS_WINDOWS);
   }
 
   @Test
-  public void testIsJavaAtLeast17() throws Exception {
+  public void testIsJavaAtLeast17() {
```

Comment:
Stupid question here: since SonarQube is now using Java8 and that classes won't be loaded by Java 7 or Java 6, does this test shouldn't be dropped ?